src/test/java/api/InstancesApiExamples.java

```diff
@@ -873,22 +879,59 @@ public void cannotRemoveHRID() throws Exception {
     assertThat(existingInstance, is(createdInstance));
   }
 
+  @Test
+  public void canFrowardInstanceCreateFailureFromStorage() throws Exception {
+    final String expectedErrorMessage = "Instance-storage is temporary unavailable for create";
+
+    instancesStorageClient.emulateFailure(new EndpointFailureDescriptor()
+      .setFailureExpireDate(DateTime.now(DateTimeZone.UTC).plusSeconds(2).toDate())
```

Comment:
Personally, I'd statically import the timezone, as I think it makes this line more readable

```suggestion
      .setFailureExpireDate(DateTime.now(UTC).plusSeconds(2).toDate())
```